pyanaconda/modules/payloads/payload/dnf/dnf.py

```diff
@@ -91,11 +102,123 @@ def process_kickstart(self, data):
         source.process_kickstart(data)
         self.add_source(source)
 
+    def _process_kickstart_packages(self, data):
+        """Process the kickstart packages."""
+        packages = PackagesConfigurationData()
+        packages.core_group_enabled = not data.packages.nocore
+        packages.default_environment_enabled = data.packages.default
+
+        if data.packages.environment is not None:
+            packages.environment = data.packages.environment
+
+        packages.packages = list(data.packages.packageList)
+        packages.excluded_packages = data.packages.excludedList
```

Comment:
Is `data.packages.excludeList` is different and does not need `list()` ?

Reply:
Fixed.